src/controllers/super-admins.js

```diff
@@ -48,7 +56,7 @@ const createSuperAdmins = (req, res) => {
       if (existingSuperAdmin) {
         return res.status(400).json({
           message: 'Error!',
-          error: 'This email exist in database for another Super Admin, please check.',
+          error: 'This email is used by another super admin.',
```

Comment:
Our backend should not give extra information when entering  an email. You should replace the error : `This email is already used`